src/paired_delimiter.py

```diff
@@ -23,26 +42,70 @@ class PairedDelimiter:
     PairedDelimiter("diamond", "angleBrackets", "<", ">"),
     PairedDelimiter("escaped quad", "escapedDoubleQuotes", '\\"', '\\"'),
     PairedDelimiter("escaped twin", "escapedSingleQuotes", "\\'", "\\'"),
+    PairedDelimiter("escaped round", "escapedParentheses", "\\(", "\\)"),
     PairedDelimiter("quad", "doubleQuotes", '"', '"'),
     PairedDelimiter("round", "parentheses", "(", ")"),
     PairedDelimiter("skis", "backtickQuotes", "`", "`"),
-    PairedDelimiter("void", "whitespace", " ", " "),
+    PairedDelimiter("void", "whitespace", " ", " ", is_selectable=False),
     PairedDelimiter("square", "squareBrackets", "[", "]"),
     PairedDelimiter("twin", "singleQuotes", "'", "'"),
+    PairedDelimiter("outside", "any", "", "", is_wrappable=False),
 ]
```

Comment:
Note how we now have surrounding pairs that can either only be used as wrappers or only as surrounding pair scope types